projects/cdk/schematics/ng-update/constants/enums.ts

```diff
@@ -0,0 +1,61 @@
+interface ReplacementEnum {
+    name: string;
+    replaceValues: Record<string, string>;
+}
+
+export const ENUMS_FOR_REPLACE: ReplacementEnum[] = [
```

Comment:
Feels like there were more enums to replace 🤔

Reply:
added more enums 👍